src/test/cypress/support/fragments/calendar-info-pane.js

```diff
@@ -0,0 +1,59 @@
+import Accordion from '@folio/stripes-testing/interactors/accordion';
+import { including } from '@interactors/html';
+import KeyValue from '@folio/stripes-testing/interactors/key-value';
+import List, { ListItem } from '@folio/stripes-testing/interactors/list';
+import Button from '../../interactors/button';
+import Pane from '../../interactors/pane';
+import Headline from '../../interactors/headline';
+import { MultiColumnListCell } from '../../interactors/multi-column-list';
+
+export const checkExpandButton = () => {
+  cy.do([
```

Comment:
Please try to make these CI errors go away

Reply:
I just pushed some changes to fix this